nuget.config

```diff
@@ -1,11 +1,10 @@
 <?xml version="1.0" encoding="utf-8"?>
 <configuration>
-  <config>
-    <add key="repositorypath" value="packages" />
-  </config>
   <packageSources>
-    <add key="api.nuget.org" value="https://api.nuget.org/v3/index.json" protocolVersion="3" />
-    <add key="vs-devcore" value="https://www.myget.org/F/vs-devcore/api/v3/index.json" />
-	  <add key="vs-impl" value="https://devdiv.pkgs.visualstudio.com/_packaging/vs-impl/nuget/v3/index.json" />
+    <clear />
+	<add key="vs-impl" value="https://devdiv.pkgs.visualstudio.com/_packaging/vs-impl/nuget/v3/index.json" />
```

Comment:
This feed requires authentication, which essentially blocks use of this repository to OSS developers. Can we use a public feed instead?

Reply:
Done